docs/BasicCommunication/OnSystemCapabilityUpdated/index.md

```diff
@@ -13,6 +13,17 @@ Purpose
 
 1. HMI sends OnSystemCapabilityUpdated notification to SDL after creating a widget _(see also [CreateWindow](../../UI/CreateWindow/index.md))_.
 2. HMI sends OnSystemCapabilityUpdated notification after receiving UI.Show(templateConfiguration) request manipulating the content presented of main window or widgets.
+3. HMI should not send `WEB_VIEW` template in `WindowCapability` for any widgets, and applications with `AppHMIType` other than `WEB_VIEW`.
+4. If the application changes template to `WEB_VIEW`, HMI should send OnSystemCapabilityUpdated with the following `textFields` restrictions in `WindowCapability`:
+    * `menuName` and `templateTitle` should be included, if they are visible on HMI.
+    * `mainField1-4`, `statusBar` and `mediaTrack` should not be included if EOM implemented the `WEB_VIEW` template without these text fields.
```

Comment:
```suggestion
    * `mainField1-4`, `statusBar` and `mediaTrack` should not be included if OEM implemented the `WEB_VIEW` template without these text fields.
```

Reply:
@kshala-ford, thank you, please, see fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/219/commits/efdae1bf4ec782ce807b778c03dd6b8d9f7ace48